File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/BrandModel.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class BrandModel {
    @SerializedName("id")
    @Expose
    public String Id;

    @SerializedName("modelName")
    @Expose
    public String ModelName;
}

File: CarsMatter.Infrastructure/Models/BrandModel.cs
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations.Schema;

namespace CarsMatter.Infrastructure.Models
{
    public class BrandModel
    {
        public int Id { get; set; }

        [JsonIgnore]
        public string HttpPath { get; set; }

        public string ModelName { get; set; }

        public int BrandId { get; set; }

        [ForeignKey("BrandId")]
        public Brand Brand { get; set; }
    }
}
